src/08_comprehensions.py

```diff
@@ -11,14 +11,17 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
+for x in [1, 2, 3, 4, 5]:
+    y.append(x)
+print(y)
 
```

Comment:
A list comprehension doesn't use a `for loop` to go through the items in the list. Here is an example for the problem above.
`[i for i in range(1,6)]`
The best way I think about it is this way when using javascript:
[`<mapping>` `<array to go through>` `<filtering>`]